Survey Form

```diff
@@ -1,35 +1,83 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-<title>Survey Form</title>
-</head>
-<body>
-    <h1 id= "title">Survey Form</h1>
-    <p id= "description">This is my survey form</p>
-    <form id="survey-form">
-      <label  id="name-label">
-<input type="text" id="name" placeholder="name" required="required ">Name</label >
-<label  id="email-label">
-<input type="email" id="email" required="required " placeholder="Email">Email</label >
-<label  id="number-label">
-<input type="number" id="number" placeholder="Number" min="10" max="20">Number</label >
-<select id="dropdown">
-  <option value="">Male
-    </option>
-    <option value="">Female
-    </option>
-    </select>
-    <input type="radio" value="male" name="white">
-    <input type="radio" value="male" name="white">
-    </input>
-    </input>
-    <input type="radio" value="Female" name="black">
-    <input type="radio" value="Female" name="black"></input>
-    </input>
-  <input type="checkbox" value="check1"></input>
-  <input type="checkbox" value="check1"></input>
-  <textarea type="textarea" id="survey-form" placeholder="comment here"></textarea>
-  <input type="submit" id="submit" value="Submit">
-    </form>
-</body>
+<!DOCTYPE.html> 
+<html>
+    <header>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Input Form</title>
+        <link rel="stylesheet" href="inputForm.css">
+    </header>
+    <body>
+    <div id="container">
+<h1 id='heading'>SIGN UP </h1>
+
+ <form>
```

Comment:
I have not put the necessary attributes for the form which are "action" and "method"